README.md

```diff
@@ -41,124 +40,129 @@ You can send metrics, histograms, or trace data from your application to the ser
+wavefront_sender = client_factory.get_client()
 ```
 
-> **Note:** When you set up a Wavefront Proxy on the specified proxy host, you specify the port it will listen to for each type of data to be sent. The `WavefrontProxyClient` must send data to the same ports that the Wavefront Proxy listens to. Consequently, the port-related parameters must specify the same port numbers as the corresponding proxy configuration properties:
+#### Initialize the WavefrontClient (CSP OAuth App)
 
-| `WavefrontProxyClient()` parameter | Corresponding property in `wavefront.conf` |
-| ----- | -------- |
-| `metrics_port` | `pushListenerPorts=` |
-| `distribution_port` | `histogramDistListenerPorts=` |
-| `tracing_port` | `traceListenerPorts=` |
+##### Prerequisites (CSP OAuth App)
 
-### Option 3: Create a WavefrontClient
-Use `WavefrontClientFactory` to create a `WavefrontClient` instance, which can send data directly to the service or send data using a Wavefront Proxy.
+* The HTTP URL of your cluster. This is the URL you connect to when you log in to the service, typically something like `http://<domain>.wavefront.com`.
+* The base HTTP URL of your CSP console. This is the URL you connect to when you log in to the CSP console, typically something like `http://console.cloud.vmware.com`.
+* Verify that you have the required permissions for adding and managing OAuth apps in this Organization. For details, see [Organization roles and permissions](https://docs.vmware.com/en/VMware-Cloud-services/services/Using-VMware-Cloud-Services/GUID-C11D3AAC-267C-4F16-A0E3-3EDF286EBE53.html#organization-roles-and-permissions-0).
+* [Create a server to server app](https://docs.vmware.com/en/VMware-Cloud-services/services/Using-VMware-Cloud-Services/GUID-327AE12A-85DB-474B-89B2-86651DF91C77.html).
 
-The `WavefrontClientFactory` supports multiple client bindings. If more than one client configuration is specified, you can create a `WavefrontMultiClient` instance, which can send data to multiple services.
-### Prerequisites
-* Sending data via Wavefront Proxy?
-  <br/>Before your application can use a `WavefrontClient` you must [set up and start a Wavefront Proxy](https://docs.wavefront.com/proxies_installing.html).
-* Sending data via direct ingestion?
-  * Verify that you have the Direct Data Ingestion permission. For details, see [Examine Groups, Roles, and Permissions](https://docs.wavefront.com/users_account_managing.html#examine-groups-roles-and-permissions).
-  * The HTTP URL of your cluster. This is the URL you connect to when you log in to the service, typically something like `http://<domain>.wavefront.com`.<br/> You can also use HTTP client with Wavefront Proxy version 7.0 or newer. Example: `http://proxy.acme.corp:2878`.
-  * [Obtain the API token](http://docs.wavefront.com/wavefront_api.html#generating-an-api-token).
+**Example**: Use a factory class to create a WavefrontClient and send data to Operations for Applications via CSP OAuth App.
```

Comment:
**Example**: Use a factory class to create a WavefrontClient and send data to Operations for Applications by using ta server to server OAuth app.